PWGHF/vertexingHF/vHFML/AliAnalysisTaskSECharmHadronvn.h

```diff
@@ -155,13 +163,15 @@ class AliAnalysisTaskSECharmHadronvn : public AliAnalysisTaskSE
 
     /// variables for ML application
     bool fApplyML;                          /// flag to enable ML application
+    bool fMultiClass = false;               /// flag to enable multi-class models (Bkg, Prompt, FD)
+
     TString fConfigPath;                    /// path to ML config file
     AliHFMLResponse* fMLResponse;           //!<! object to handle ML response
-    int fNMLBins;                           /// number of bins for ML axis in thnsparse
-    double fMLOutputMin;                    /// minimum value of ML ouptut in thnsparse axis
-    double fMLOutputMax;                    /// maximum value of ML ouptut in thnsparse axis
+    int fNMLBins[3] = {1000, 100, 100};     /// number of bins for ML output axis in THnSparse
```

Comment:
Since you initialise these data members here, you could move also all the others (not necessary for this PR, but it would be good to have the same for all the datamembers)